<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearProducto</class>
 <widget class="QDialog" name="crearProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}

QPushButton{
	background:#0069BF;
	border-radius:10px;
	color:white;
}

QPushButton:hover{
	background:#008EFF;
}

QLineEdit{
	border-radius:14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#0069BF;
color: white;</string>
     </property>
     <property name="text">
      <string>Crear Producto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Marca:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leMarca"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Modelo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leModelo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Precio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePrecio"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Descripcion:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leDescripcion"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnCrearProd">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCrearProd</sender>
   <signal>clicked()</signal>
   <receiver>crearProducto</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
